### PR TITLE
Updating file paths and other issues with windows environment

### PR DESCRIPTION
**Pull request by harshpalan**
Creating a PR to update issues with test cases and other files while building for the Windows environment.

### CODE REVIEW

File: mirdata/datasets/compmusic_jingju_acappella.py [hunk 1]
```diff
@@ -281,7 +281,7 @@ def load_phrases(fhandle: TextIO) -> annotations.LyricData:
     start_times = []
     end_times = []
     lyrics = []
-
+    # fhandle.reconfigure(encoding="utf-8")
```
Comment (magdalenafuentes) [inline mirdata/datasets/compmusic_jingju_acappella.py:284]: Remove comment

File: mirdata/datasets/compmusic_jingju_acappella.py [hunk 2]
```diff
@@ -308,7 +308,7 @@ def load_syllable(fhandle: TextIO) -> annotations.LyricData:
     start_times = []
     end_times = []
     events = []
-
+    # fhandle.reconfigure(encoding="utf-8")
```
Comment (magdalenafuentes) [inline mirdata/datasets/compmusic_jingju_acappella.py:311]: Remove comment

File: mirdata/datasets/saraga_hindustani.py [hunk 3]
```diff
@@ -379,7 +379,7 @@ def load_sections(fhandle):
     """
     intervals = []
     section_labels = []
-
+    # fhandle.reconfigure(encoding="utf-8")
```
Comment (magdalenafuentes) [inline mirdata/datasets/saraga_hindustani.py:382]: remove comment

File: tests/datasets/test_dali.py [hunk 4]
```diff
@@ -1,4 +1,14 @@
-import DALI
+import logging
+import os
+
+try:
+    import DALI
+except ImportError:
+    logging.error(
+        "In order to use dali you must have dali-dataset installed. "
```
Comment (magdalenafuentes) [inline tests/datasets/test_dali.py:8]: Maybe "In order to `test` dali ..."?

File: tests/test_io.py [hunk 5]
```diff
@@ -68,7 +68,7 @@ def func(fh):
 
 
 def test_coerce_to_string_io_with_path():
-    with tempfile.NamedTemporaryFile() as f:
+    with tempfile.NamedTemporaryFile(delete=False) as f:
```
Comment (magdalenafuentes) [inline tests/test_io.py:71]: Why this change?

**Review by magdalenafuentes — CHANGES_REQUESTED** [review]
@harshpalan I know you're finishing this up, but fantastic job so far! 

See my comments and let me know if anything's not clear

File: mirdata/datasets/compmusic_jingju_acappella.py (hunk 2 above)
Comment (harshpalan) [reply mirdata/datasets/compmusic_jingju_acappella.py:311]: Updated

File: tests/test_io.py (hunk 5 above)
Comment (harshpalan) [reply tests/test_io.py:71]: The file was getting deleted as soon as it was closed and thus throwing an error so had to pass `delete=False` to avoid that.

**Review by magdalenafuentes — APPROVED** [review]
Fantastic job @harshpalan 🎉 

Feel free to merge this!
